src/app/components/core/configuration-manager/show-configuration/show-configuration.component.html

```diff
@@ -39,11 +39,23 @@
                       [type]="configuration.type" class="input is-fullwidth is-small" [minValue]="configuration.minimum"
                       [maxValue]="configuration.maximum" appNumberOnly />
 
-                    <ngx-codemirror *ngSwitchCase="'JSON'" [formControlName]="configuration.key" #jsonEditor
-                      [ngClass]="{'readonly-editor': form.controls[configuration.key]?.status === 'DISABLED' }"
-                      [options]="setCodeMirrorOption(configuration)" [autoFocus]="true"
-                      [jsonValue]="form.controls[configuration.key]">
-                    </ngx-codemirror>
+                    <ng-container *ngSwitchCase="'JSON'">
+                      <ngx-codemirror [formControlName]="configuration.key" #jsonEditor
+                        [ngClass]="{'readonly-editor': form.controls[configuration.key]?.status === 'DISABLED' }"
+                        [options]="setCodeMirrorOption(configuration)" [autoFocus]="true"
+                        [jsonValue]="form.controls[configuration.key]">
+                      </ngx-codemirror>
+                      <p class="help" *ngIf="configuration?.fileName">
```

Comment:
This element could be removed as no file is being created on server, so filename property does not exist for JSON type config item.